serve/mlc_serve/utils.py

```diff
@@ -25,7 +25,6 @@ def get_default_mlc_serve_argparser(description="", allow_override=False):
     parser.add_argument("--use-sync-engine", action="store_true")
     parser.add_argument("--num-sequences-to-sample", type=int, default=1)
     parser.add_argument("--max-num-batched-tokens", type=int, default=4096)
-    parser.add_argument("--num-sequences-to-sample", type=int, default=1)
```

Comment:
Hi, is there any specific reason for this removal?

Reply:
Yes, it is duplicate of line 26

Reply:
Got it.